desktop/apps/consignments2/client/actions.js

```diff
@@ -125,6 +135,7 @@ export function completeSubmission () {
       dispatch(updateSubmission(submissionResponse.body))
       dispatch(push('/consign2/submission/thank_you'))
     } catch (err) {
+      dispatch(stopLoading())
       dispatch(updateError('Unable to submit at this time.'))
       console.error('error!', err)
```

Comment:
Do you need this `console.log`? Just an in-general suggestion, but if you're using it for logging purposes it might be good to make it a bit more verbose since Force's compiled source-map support is non-existent which makes tracking down errors more difficult than need be. Something like 
```
console.error(
  '(consignments/client/actions.js @ completeSubmission) Error:', err
)